exporter/tanzuobservabilityexporter/metrics_test.go

```diff
@@ -268,7 +270,7 @@ func TestGaugeConsumerMissingValue(t *testing.T) {
 }
 
 func createMetricInfo(metric pdata.Metric) metricInfo {
-	mi := metricInfo{Metric: metric, Source: "test_source"}
+	mi := metricInfo{Metric: metric, Source: "test_source", SourceKey: "host.name"}
```

Comment:
I think we should inline this helper so that it's easy to see the hard-coded string values that are later used as assertions in the test.